File: c/1008.c

// https://www.urionlinejudge.com.br/judge/pt/problems/view/1008

#include <stdio.h>
#include <stdlib.h>

int main()
{
    int fun, horas;
    double pag;

    scanf("%d %d %lf", &fun, &horas, &pag);

    printf("NUMBER = %d\n", fun);
    printf("SALARY = U$ %.2lf\n", pag * horas);

    return 0;
}

File: haskell/1008.hs

-- https://www.urionlinejudge.com.br/judge/pt/problems/view/1008

import Text.Printf (printf)

main :: IO ()
main = do id <- readLn :: IO Int
          horas <- readLn :: IO Float
          salarioHora <- readLn :: IO Float
          printf "NUMBER = %d\n" id
          printf "SALARY = U$ %.2f\n" (horas * salarioHora)